pkg/exporter/jobs.go

```diff
@@ -1,14 +1,17 @@
 package exporter
 
 import (
+	"bytes"
+	"fmt"
 	"reflect"
 	"regexp"
+	"strconv"
 
 	"github.com/mvisonneau/gitlab-ci-pipelines-exporter/pkg/schemas"
 	log "github.com/sirupsen/logrus"
 )
 
-func pullRefPipelineJobsMetrics(ref schemas.Ref) error {
+func pullRefPipelineJobsMetrics(ref schemas.Ref, pullJobTraces bool) error {
```

Comment:
could we set pullJobTraces as part of the ref to avoid overloading the subsequent functions?

Reply:
Would also be glad about it - but I've no good idea how to implement this - or golang experience.
We have two separate schedules for normal and "enriched" metrics (with traces). Both of them trigger the same metric fetch mechanism - with the **pullJobTraces** flag set to **false/true**. If that would be a part of **ref**, we have to switch it there every time.
Another idea - is it possible to define the flag on the scheduler itself and get the value in the **exporter/job.go** ?
I suppose I need your help here :)